webpack/assets/javascripts/react_app/redux/API/API.js

```diff
@@ -13,28 +12,28 @@ axios.defaults.headers.common['X-CSRF-Token'] = getcsrfToken();
 
 export default {
```

Comment:
If this doesn't call `foremanUrl`, what's the value of this over directly using axios? That it's easier to replace the implementation?

Reply:
I'd say so. It's easier to replace implementation, but given we have middleware that takes care of all the API, we already have middle layer, that where we could call the appropriate library. So there is probably no point to it :thinking:

Reply:
Please notice that the middleware still doesn't support all the operations (added in this pr: https://github.com/theforeman/foreman/pull/7261) so until that gets in I think we should keep the `foremanUrl` here. Also if you delete it here this PR should add it to all places where api is used with a custom url

Reply:
Right, but once we support them all in middleware, this is loosing the additional value I gues. This PR definitelly needs to add it to all places where we use custom URL :) but as of now, we have no option to turn off the prefix usage for backend generated URLs (what's a majority of URLs as of now)

Reply:
It does feel redundant once we have the middleware, but the middleware only works when you are performing a redux action.

Would we ever want to support API calls outside of the redux scope?